polkadot/xcm/xcm-executor/src/traits/should_execute.rs

```diff
@@ -127,3 +127,65 @@ impl CheckSuspension for Tuple {
 		false
 	}
 }
+
+/// Trait to determine whether the execution engine should not execute a given XCM.
+///
+/// Can be amalgamated into a tuple to have multiple trials. If any of the tuple elements returns
+/// `Err(())`, the execution stops. Else, `Ok(_)` is returned if all elements accept the message.
+pub trait ShouldNotExecute {
+	/// Returns `Ok(())` if the given `message` may be executed.
```

Comment:
I think this description is copied from `ShouldExecute`. Please revise it to something like: "`Ok(())` means there is no reason not to execute, while `Err(e)` indicates there is a reason not to execute."

Reply:
https://github.com/paritytech/polkadot-sdk/pull/7169/commits/e10587b09a24c6537bb2ca17f8d602c253711e4b